### PR TITLE
Feature/[Ubs order] 1 Tab seperate functional (Bonus and Certificate)

### PR DESCRIPTION
**Pull request by Flash5482**

### CODE REVIEW

**Review by NewArtMask — APPROVED** [review]

File: src/app/main/component/ubs/components/ubs-order-details/ubs-order-certificate/ubs-order-certificate.component.ts [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+    });
+    this.showTotal = this.total;
+    if (this.total < this.minOrderValue && this.orderDetailsForm.dirty) {
+      this.displayMinOrderMes = true;
+      this.onSubmit = true;
+    } else {
+      this.displayMinOrderMes = false;
+      this.onSubmit = false;
+    }
+    this.finalSum = this.total - this.pointsUsed;
```
Comment (hnativlyubomyr) [inline src/app/main/component/ubs/components/ubs-order-details/ubs-order-certificate/ubs-order-certificate.component.ts:138]: please, replace if-else:
this.displayMinOrderMes = this.total < this.minOrderValue && this.orderDetailsForm.dirty;
this.onSubmit = this.displayMinOrderMes

File: src/app/main/component/ubs/components/ubs-order-details/ubs-order-certificate/ubs-order-certificate.component.ts [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+  }
+
+  private clearAdditionalCertificate(index: number) {
+    if (this.formArrayCertificates.length > 1) {
+      if (this.certificates.length === 0) {
+        this.certificateReset(true);
+      }
+      this.formArrayCertificates.removeAt(index);
+    } else {
+      this.certificateReset(true);
```
Comment (hnativlyubomyr) [inline src/app/main/component/ubs/components/ubs-order-details/ubs-order-certificate/ubs-order-certificate.component.ts:249]: duplicate code
move  this.certificateReset(true) before if

File: src/app/main/component/ubs/components/ubs-order-details/ubs-order-certificate/ubs-order-certificate.component.ts [hunk 3]
```diff
@@ -0,0 +1,367 @@
...
+    this.shareFormService.orderDetails.certificates = this.certificates;
+    this.shareFormService.orderDetails.certificatesSum = this.showCertificateUsed;
+    this.shareFormService.orderDetails.pointsSum = this.pointsUsed;
+    this.shareFormService.orderDetails.total = this.showTotal;
+    this.shareFormService.orderDetails.finalSum = this.finalSum;
+    this.shareFormService.changeOrderDetails();
+  }
+
+  calculateCertificates(arr): void {
+    console.log(arr);
```
Comment (hnativlyubomyr) [inline src/app/main/component/ubs/components/ubs-order-details/ubs-order-certificate/ubs-order-certificate.component.ts:173]: remove console.log

File: src/app/main/component/ubs/components/ubs-order-details/ubs-order-details.component.ts [hunk 4]
```diff
@@ -80,6 +79,7 @@ export class UBSOrderDetailsComponent extends FormBaseComponent implements OnIni
   public currentLocation: string;
   public isFetching = false;
   public changeLocation = false;
+  isBonus: any;
```
Comment (hnativlyubomyr) [inline src/app/main/component/ubs/components/ubs-order-details/ubs-order-details.component.ts:82]: why any?  it's boolean

File: src/app/main/component/ubs/components/ubs-order-details/ubs-order-certificate/ubs-order-certificate.component.ts (hunk 1 above)
Comment (Flash5482) [reply src/app/main/component/ubs/components/ubs-order-details/ubs-order-certificate/ubs-order-certificate.component.ts:133]: done

**Review by BobNazar — APPROVED** [review]

**Review by JohnSteck9 — APPROVED** [review]

**Review by Yura935 — APPROVED** [review]

**Review by KhrystynaPr — APPROVED** [review]

**Review by ILyaDyakun — APPROVED** [review]

**Pull request merged**